### DISCUSSION TITLE
low correlation between scATAC and scRNA

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#1018] [Q&A] [answered]
Hi Tim,
I have integrated my scATAC datasets and predicted its cell labels by  integrating it with the scRNA dataset following https://satijalab.org/signac/articles/pbmc_vignette.html. The prediction scores looks pretty good for each cell type and the highest prediction scores are all above 0.7 in most of the cases.

However, when I tried to manually calculate the correlation between scATAC and scRNA in the way below, the correlation between any pariwise cell type is extremely low which is not consistent with good results in the prediction scores:

**what I did is like this:**

(1) subset each cell type of RNA assay of **scATAC** (this RNA assay was created by estimating gene activity) and identify variable genes by running FindVariableFeatures (lets call it **var.genes.ATAC**) 
(2) subset each cell type of RNA assay of **scRNA** and identify variable genes by running FindVariableFeatures (lets call it **var.genes.RNA**) 
(3) take the intersection of **var.genes.ATAC** and **var.genes.RNA** and name it as **common.genes**
(4) take the data slot from the RNA assay of both scRNA and scATAC and only keep the **common.genes**. name the shortened version of the data from the **data** slot as **RNA_data_ATAC_new** and **RNA_data_new**, respectively
(5)  calculate the correlation by running:   
 
   ```
avg_RNA_data_ATAC_new <- rowMeans(RNA_data_ATAC_new)#average of each gene in this cell type
   avg_RNA_data_new <- rowMeans(RNA_data_new)# #average of each gene in this cell type
 correlation <- cor(avg_RNA_data_ATAC_new, avg_RNA_data_new, method = "pearson")

```

However, like i said, the correlation between each cell type paisewise is extremely low and whats worse,the same cell type correlation is even lower than other pairwise correlation which makes me so confused and also sad. Can you give me some suggestions on whats going on? Why there are high prediction scores but such low correlation scores? whatis the difference between my calculation and yours (like anchors identification etc)? 

![image](https://user-images.githubusercontent.com/54915282/156620650-8d4f918f-4c0f-4753-93f7-1108f4cc4732.png)

**Comment by timoast** [comment, marked as answer]
See https://github.com/timoast/signac/discussions/1026
